glyphs-reader/src/font.rs

```diff
@@ -764,24 +764,24 @@ impl FontMaster {
             .and_then(|metric| metric.pos)
     }
 
-    pub fn ascender(&self) -> Option<OrderedFloat<f64>> {
-        self.read_metric("ascender")
+    pub fn ascender(&self) -> Option<f64> {
+        self.read_metric("ascender").map(|x| x.into_inner())
     }
 
-    pub fn descender(&self) -> Option<OrderedFloat<f64>> {
-        self.read_metric("descender")
+    pub fn descender(&self) -> Option<f64> {
+        self.read_metric("descender").map(|x| x.into_inner())
     }
 
-    pub fn x_height(&self) -> Option<OrderedFloat<f64>> {
-        self.read_metric("x-height")
+    pub fn x_height(&self) -> Option<f64> {
+        self.read_metric("x-height").map(|x| x.into_inner())
     }
 
-    pub fn cap_height(&self) -> Option<OrderedFloat<f64>> {
-        self.read_metric("cap height")
+    pub fn cap_height(&self) -> Option<f64> {
+        self.read_metric("cap height").map(|x| x.into_inner())
     }
 
-    pub fn italic_angle(&self) -> Option<OrderedFloat<f64>> {
-        self.read_metric("italic angle")
+    pub fn italic_angle(&self) -> Option<f64> {
+        self.read_metric("italic angle").map(|x| x.into_inner())
```

Comment:
Optional, this repeats enough I'd be tempted to add a helper. self.read_metric_f64("italic angle") or some such.

Or just update the read_metric fn to return f64 if that wouldn't break the world?